cPRONTO/oppni.py

```diff
@@ -1611,15 +1670,28 @@ def run_jobs(job_paths, run_locally, num_procs, depends_on_step):
             # if ret_code < 0 or ret_code > 0:
             #     raise OSError('Error executing the {} job locally.'.format(step_id))
 
-            # list of all script paths
-            jobs_list = job_paths.values()
+            #removed Jun 26 2018
+            #jobs_list = job_paths.values()
 
+            #added Jun 26 2018
+            jobs_list = list(job_paths.values())
```

Comment:
lets look into this - this must be a list

Reply:
inside run_jobs(job_paths...) called from line 1620 = run_jobs(jobs_dict..) with dictionary passed in